src/jit/lsraxarch.cpp

```diff
@@ -2382,6 +2383,19 @@ void LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
             break;
         }
 
+        case NI_SSE41_Extract:
+            if (baseType == TYP_FLOAT)
+            {
+                info->internalIntCount += 1;
```

Comment:
Should we explicitly set the internal candidates, as we have done elsewhere?

Reply:
No necessary, this internal register can be any GPR.